Arreglar test de "accrualPeriodicity" después de permitir el caso de "EVENTUAL" en mayúsculas

Se agregó como un valor permitido el caso de "EVENTUAL" en mayúsculas, pero es difícil encontrar dónde y como cambiar el test que falla para que esté actualizado a la nueva situación.





.......................................................................................................................................................................................................F............
======================================================================
FAIL: tests.test_validation.TestDataJsonTestCase.test_validity(u'malformed_accrualperiodicity', {u'status': u'ERROR', u'error': {u'catalog': {u'status': u'OK', u'errors': [], u'title': u'Datos Argentina'}, u'dataset': [{u'status': u'ERROR', u'identifier': u'99db6631-d1c9-470b-a73e-c62daa32c420', u'list_index': 0, u'errors': [{u'instance': u'RP1Y', u'validator': u'anyOf', u'path': [u'dataset', 0, u'accrualPeriodicity'], u'message': u"u'RP1Y' is not valid under any of the given schemas", u'error_code': 2, u'validator_value': [{u'pattern': u'^R/P\\d+(\\.\\d+)?[Y|M|W|D]$', u'type': u'string'}, {u'pattern': u'^R/PT\\d+(\\.\\d+)?[H|M|S]$', u'type': u'string'}, {u'pattern': u'^eventual$', u'type': u'string'}]}], u'title': u'Sistema de contrataciones electr\xf3nicas'}]}})
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/abenassi/anaconda/envs/pydatajson/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/abenassi/github/pydatajson/tests/test_validation.py", line 76, in run_case
    assert_dict_equal(expected_dict, response_dict)
AssertionError: {u'status': u'ERROR', u'error': {u'catalog': {u'status': u'OK', u'errors': [], u [truncated]... != {u'status': u'ERROR', u'error': {u'catalog': {u'status': u'OK', u'errors': [], u [truncated]...
Diff is 1821 characters long. Set self.maxDiff to None to see it.
-------------------- >> begin captured stdout << ---------------------
{
    "status": "ERROR",
    "error": {
        "catalog": {
            "status": "OK",
            "errors": [],
            "title": "Datos Argentina"
        },
        "dataset": [
            {
                "status": "ERROR",
                "identifier": "99db6631-d1c9-470b-a73e-c62daa32c420",
                "list_index": 0,
                "errors": [
                    {
                        "instance": "RP1Y",
                        "validator": "anyOf",
                        "path": [
                            "dataset",
                            0,
                            "accrualPeriodicity"
                        ],
                        "message": "u'RP1Y' is not valid under any of the given schemas",
                        "error_code": 2,
                        "validator_value": [
                            {
                                "pattern": "^R/P\\d+(\\.\\d+)?[Y|M|W|D]$",
                                "type": "string"
                            },
                            {
                                "pattern": "^R/PT\\d+(\\.\\d+)?[H|M|S]$",
                                "type": "string"
                            },
                            {
                                "pattern": "^eventual$",
                                "type": "string"
                            },
                            {
                                "pattern": "^EVENTUAL$",
                                "type": "string"
                            }
                        ]
                    }
                ],
                "title": "Sistema de contrataciones electr\xf3nicas"
            }
        ]
    }
}

--------------------- >> end captured stdout << ----------------------

----------------------------------------------------------------------
Ran 212 tests in 33.347s

FAILED (failures=1)

--- a/tests/support/factories/dataset_errors.py
+++ b/tests/support/factories/dataset_errors.py
@@ -30,7 +30,11 @@ def dataset_error(options=None):
             {
                 "pattern": "^eventual$",
                 "type": "string"
-            }
+            },
+            {
+                "pattern": "^EVENTUAL$",
+                "type": "string"
+            },
         ]
     }
     if options is not None:

--- a/tests/test_validation.py
+++ b/tests/test_validation.py
@@ -78,7 +78,6 @@ class TestDataJsonTestCase(object):
     # Tests de CAMPOS REQUERIDOS
 
     # Tests de inputs válidos
-    @nottest
     def test_validity(self):
         for filename, value_or_none in iteritems(TEST_FILE_RESPONSES):
             yield self.run_case, filename, value_or_none

